js/Modules/newModule.js

```diff
@@ -1,20 +1,30 @@
 interviewApp.Modules.NewModule = {
-    init: function(id=''){
+    init(id = '') {
         const isIdSet = !!id;
-        
+
+
+        const hxttpNew = new XMLHttpRequest();
+        hxttpNew.onreadystatechange = function () {
+            if (this.readyState == 4 && this.status == 200) {
+                // Typical action to be performed when the document is ready:
+
+                const newObj = JSON.parse(hxttpNew.responseText);
```

Comment:
Same comment about potential errors thrown by `JSON.parse` on non-conforming responses as somewhere above. Consider wrapping in a `try` / `catch` block.